File: java/esfera.java
import java.util.Scanner;
public class esfera {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double total = sc.nextDouble();
        total *= total * total;
        total *= (4.0/3.0) * 3.14159;
        String str = String.format("%.3f",total);
        System.out.println("VOLUME = " + str);
        sc.close();
    }
}

File: apenas java/esfera.c
#include <stdio.h>
#define PI 3.14159
int main() {
    double total;
    scanf("%lf",&total);
    total *= total * total;
    total *= (4.0/3.0) * PI;
    printf("VOLUME = %.3f\n",total); 
    return 0;
}
